packages/terra-form-select/src/Select.jsx

```diff
@@ -240,10 +247,19 @@ class Select extends React.Component {
       });
     });
 
+    // If there is no selected option, or none of the options match the selected value, create the default option.
     if (!selectedValue) {
       display = intl.formatMessage({ id: 'Terra.form.select.defaultDisplay' });
       selectedValue = '';
-      defaultOption = <SelectOption value={selectedValue} display={display} tabIndex="-1" isSelected className={cx('default-option')} />;
+      defaultOption = (
```

Comment:
Did you mean to drop `tabIndex="-1"`?

Reply:
Yeah, not allowing focus made keyboard navigation a little weird and wasn't necessary for functionality so I dropped it.